docs/_posts/2024/2024-03-31-scala-io-nantes.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: "post"
+author:
+  - "pauline"
+title: "Duchess at Scala.io"
+date: "2024-04-02"
+categories:
+  - "code"
+  - "event"
+---
+
+| ![Duchess in Scala.io](/assets/sponsors/scalaio.webp){: width="300"} | Duchess was at Scala.io for the 2024 edition in Nantes |
+
+
+A couple of weeks ago was the scala conference in Nantes. This article focus of the scala women speakers present at the event.
+Also Scala.io organizers offered 3 tickets to the Duchess France community.
+
+# Focus on Monica's talk
+
+Great feedback from Monica. She shared with us about her first year of java and gave very good hints.
```

Comment:
Proposition de formulation : "Monica shared with us valuable knowledge that she acquired while learning Scala for the first time".